recipes/simplification/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "simplification" %}
+{% set version = "0.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/urschrei/simplification/archive/v{{ version }}.tar.gz
+  sha256: b056fa0ca0cd05f9f7c2303f382d511a2c1f3c116511a097d9f223c950ad180d
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
```

Comment:
Can you clear out the template comments here and below?

Reply:
Just left them as a reminder for things I am not sure how to handle and looking for advise, to make sure these are handled...